### PR TITLE
Stage 1 - dependency upgrade

### PR DESCRIPTION
**Pull request by wbrunette**

### CODE REVIEW

**Comment by wbrunette** [comment]
runtests

**Comment by wbrunette** [comment]
runtests

**Comment by wbrunette** [comment]
runtests

File: survey_app/src/androidTest/java/org/opendatakit/survey/BasicTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import static org.junit.Assert.assertTrue;
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class BasicTest {
+
+    @Test
+    public void testFailureUsedToVerifyBuildsCatchWhenSet() {
+        //assertTrue(false);
+        assertTrue(true);
```
Comment (jbeorse) [inline survey_app/src/androidTest/java/org/opendatakit/survey/BasicTest.java:16]: What purpose is this serving? The name implies it was meant to fail, but now it seems like its meant to pass as a no-op? Should we rename? Or remove?

File: survey_app/src/main/java/org/opendatakit/survey/application/Survey.java [hunk 2]
```diff
@@ -37,14 +39,20 @@ public static Survey getInstance() {
     return singleton;
   }
 
+  @Override
+  protected void attachBaseContext(Context base) {
+    super.attachBaseContext(base);
+    MultiDex.install(this);
+  }
+
+
```
Comment (jbeorse) [inline survey_app/src/main/java/org/opendatakit/survey/application/Survey.java:48]: Nit: Extra whitespace

**Review by jbeorse — APPROVED** [review]
Couple nits but looks good

**Pull request merged**